app/src/main/java/com/example/androidproject/recycler/RecyclerAdapter.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.androidproject.recycler
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.androidproject.R
+import kotlinx.android.synthetic.main.list_item_layout.view.*
+
+class RecyclerAdapter(private val itemList: List<Int>) : RecyclerView.Adapter<RecyclerAdapter.ListViewHolder>() {
+    override fun getItemCount(): Int {
```

Comment:
```suggestion
    override fun getItemCount(): Int = itemList.size
```
Always use inline function response where possible